--- a/configure.ac
+++ b/configure.ac
@@ -63,7 +63,7 @@
     GTKUNIXPRINT_PC="gtk+-unix-print-2.0"
     ;;
   3.0)
-    DEPENDENCY_PC="glib-2.0 >= $GLIB_REQUIRED gtk+-3.0 >= $GTK3_REQUIRED webkitgtk-3.0 >= $WEBKIT_REQUIRED"
+    DEPENDENCY_PC="glib-2.0 >= $GLIB_REQUIRED gtk+-3.0 >= $GTK3_REQUIRED "
     GTKUNIXPRINT_PC="gtk+-unix-print-3.0"
     AC_DEFINE([HAVE_GNOME3], 1,
               [Use GNOME 3 technologies])

--- a/src/gnome-web-photo.c
+++ b/src/gnome-web-photo.c
@@ -28,7 +28,7 @@
 #include <glib.h>
 #include <glib/gi18n.h>
 #include <gtk/gtk.h>
-#include <webkit/webkit.h>
+#include <webkit2/webkit2.h>
 
 #ifdef HAVE_GTK_UNIX_PRINT
 #include <gtk/gtkunixprint.h>
--- a/autogen.sh
+++ b/autogen.sh
@@ -3,8 +3,6 @@
 
 srcdir=`dirname $0`
 test -z "$srcdir" && srcdir=.
-
-PKG_NAME="gnome-web-photo"
 
 (test -f $srcdir/src/gnome-web-photo.c) || {
     echo -n "**Error**: Directory "\`$srcdir\'" does not look like the"
